Client/Frontend/Browser/Favicons/FaviconFetcher.swift

```diff
@@ -410,12 +410,19 @@ class FaviconFetcher {
     
     /// Determines if the downloaded image should be padded because its edges
     /// are for the most part transparent
-    private func isIconBackgroundTransparentAroundEdges(_ icon: UIImage, completion: @escaping (_ isTransparent: Bool) -> Void) {
+    static func isIconBackgroundTransparentAroundEdges(_ icon: UIImage, completion: @escaping (_ isTransparent: Bool) -> Void) {
+        if icon.size.width.isZero || icon.size.height.isZero {
+            completion(false)
```

Comment:
Maybe put on main thread too? Since this is called from `downloadIcon` which I assume is async

Reply:
I think downloadIcon comes back on main thread, but I fixed this anyways